src/test/java/plugins/SshSlavesPluginTest.java

```diff
@@ -123,7 +127,7 @@
         
         
         verify();
-        verifyLog("sh -c \"cd \"/tmp/" + slave.getName() + "\" && java  -jar slave.jar\"");
+        verifyLog("sh -c \"cd \"" + REMOTE_FS + "\" && java  -jar slave.jar\"");
```

Comment:
We need to keep the slave name in the path to prevent several slaves using the same directory at the same time.